frontend/src/components/pages/RegistrantExperience/RegistrationSteps/AdditionalInfo/QuestionCards/CamperQuestionsCard.tsx

```diff
@@ -29,9 +29,9 @@ const CamperQuestionsCard = ({
     <QuestionsCardWrapper title={`${camper.firstName} ${camper.lastName}`}>
       <VStack py="24px">
         <Wrap width="100%" px="20px" justify="space-between">
-          {campSpecificFormQuestions.map((question) => (
+          {campSpecificFormQuestions.map((question, questionIndex) => (
```

Comment:
tejas updated this in his bug fix PR (which I think we might merge before this PR). You might need to update this later due to merge conflicts